cdap-ui/server/express.js

```diff
@@ -315,7 +315,10 @@ function makeApp (authAddress, cdapConfig, {uiSettings, uiThemeConfig}) {
   ]);
   // serve static assets
   app.use('/assets', [
-    express.static(DIST_PATH + '/assets'),
+    express.static(DIST_PATH + '/assets', {
```

Comment:
since we have the `process.env.NODE_ENV === 'production'`, is it worth creating an if/else statement here to not cache if we're not on production?